docs/user/installation_and_configuration.rst

```diff
@@ -44,6 +44,11 @@ Versions and Downloads
 
 * Source: https://github.com/locationtech/geomesa/archive/master.tar.gz
 
+**Latest Eclipse IP Reviewed release**: |eclipse_release|
```

Comment:
Can we change this to 'Fully Eclipse IP Reviewed'? And maybe also add a note that you should only use this if you are very worried about licensing issues...